app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+
+  def show
+    @user = User.find params[:id]
```

Comment:
Trong trường hợp không tìm thấy, có thể handle exception như sau:

def show
    begin
      @user = User.find params[:id]
    rescue ActiveRecord::RecordNotFound
      flash[:danger] = "User does not exist"
      redirect_to root_path
    end
  end